tests/std/tests/P2443R1_views_chunk_by_death/test.cpp

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#define _CONTAINER_DEBUG_LEVEL 1
+
+#include <cassert>
+#include <cstddef>
+#include <ranges>
+#include <vector>
+
+#include <test_death.hpp>
+using namespace std;
+
+static int some_ints[] = {0, 1, 2, 3};
+
+struct test_predicate {
+    struct tag {};
+
+    test_predicate() = default;
+    test_predicate(const test_predicate&) {
+        throw tag{};
+    }
+    test_predicate& operator=(const test_predicate&) = delete;
+
+    constexpr bool operator()(int i, int j) const {
+        return i == j;
+    }
+};
+
+auto with_no_predicate() {
+    using V = decltype(ranges::chunk_by_view{some_ints, test_predicate{}});
+    V r{some_ints, {}};
+    try {
+        r = V{some_ints, {}};
+    } catch (const test_predicate::tag&) {
+    }
+    return r;
+}
+
+void test_view_predicate() {
+    auto r = with_no_predicate();
+    (void) r.pred(); // chunk_by_view has no predicate
+}
+
+void test_view_begin() {
+    auto r = with_no_predicate();
+    (void) r.begin(); // cannot call begin on a chunk_by_view that holds no predicate
+}
+
+constexpr auto lambda = [](int i, int j) { return i == j; };
+using FV              = decltype(ranges::chunk_by_view{some_ints, lambda});

```

Comment:
This was copy-pasted from:
https://github.com/microsoft/STL/blob/afe0800cd6bf0d163bf678ad4e1feb3c130d1ec5/tests/std/tests/P0896R4_views_filter_death/test.cpp#L51
The abbreviation `FV` should be changed.